lib/fuel-auth.js

```diff
@@ -99,13 +106,23 @@ module.exports = class FuelAuth {
 	}
 	_requestToken(requestOptions) {
 		// set auth options for request
+
+		const payload = {};
+		if(this.apiVersion === 2){
+			payload.client_id = this.clientId;
+			payload.client_secret = this.clientSecret;
+			payload.grant_type = 'client_credentials';
+			payload.account_id = this.accountId;
```

Comment:
account_id and scope should be removed as its not mandatory. At the FuelSDK-Node end, user wouldn't pass these information. If you send scope as null or empty, The token will be returned with no authorization and user won't be able to perform the needed operation.

Reply:
account_id is needed to enable connecting to the correct business unit